### DISCUSSION TITLE
How to chage color of selected NavigationStore?

### DISCUSSION THREAD

**Discussion by IlyaKotomin** [lepoco/wpfui#552] [Q&A] [answered]
Hello, im trying to change color, but i dont know how to do it.
I want to do like in the Radiograph
![image](https://user-images.githubusercontent.com/112555224/222978253-2b50e374-815b-41c0-806f-d1cd9e08141d.png)
![image](https://user-images.githubusercontent.com/112555224/222978195-c6edb4fb-d883-4367-9178-bbc9d0a92285.png)

**Comment by Aybex** [comment]
I think by default its defined by current Windows theme/accent color

**Reply by IlyaKotomin** [reply]
Yes it is, but i want to do like in Radiograph, there is green color

**Comment by kevfrench** [comment, marked as answer]
You can override the accent color for your app with:

```c#
var accent = System.Windows.Media.Color.FromRgb(0x0A, 0x9A, 0x36);
Wpf.Ui.Appearance.Accent.Apply(accent);
```

Or if you only want to change the Navigation View, you can override the color used with a local resource block:

```xaml
<ui:NavigationView ...>
  <ui:NavigationView.Resources>
    <Color x:Key="SystemAccentColorSecondary">#0A9A36</Color>
  </ui:NavigationView.Resources>
  <!--  Nav Content -->
</ui:NavigationView>
```

**Reply by Chrlss** [reply]
How

**Comment by SamKr** [comment]
In version 3, this looks like to be done by using `ApplicationAccentColorManager.Apply`, but I can't get that to work properly. After changing theme, it defaults to Windows' accent color, even if I re-run it. Anyone has any experience with this?

Edit: nevermind, internal code issue.

**Reply by GalaxyGaming2000** [reply]
What did you end up having to do im stuck here lol

**Reply by SamKr** [reply]
The underlying issue isn't fixed unfortunately: https://github.com/lepoco/wpfui/issues/965

**Reply by Chrlss** [reply]
How
